src/qml/pages/node/PeerDetails.qml

```diff
@@ -0,0 +1,286 @@
+// Copyright (c) 2024 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import org.bitcoincore.qt 1.0
+import "../../controls"
+import "../../components"
+
+Page {
+    id: root
+    signal backClicked()
+
+    property PeerDetailsModel details
+
+    Connections {
+        target: details
+        function onDisconnected() {
+            root.backClicked()
+        }
+    }
+
+    background: null
+    header: NavigationBar2 {
+        leftItem: NavButton {
+            iconSource: "image://images/caret-left"
+            text: qsTr("Back")
+            onClicked: root.backClicked()
+        }
+        centerItem: Header {
+            headerBold: true
+            headerSize: 18
+            header: qsTr("Peer " + details.nodeId)
+        }
+    }
+
+    ScrollView {
+        id: scrollView
+        width: parent.width
+        height: parent.height
+        clip: true
+        contentWidth: width
+
+        Column {
+            width: Math.min(parent.width - 40, 450)
+            anchors.horizontalCenter: parent.horizontalCenter
+            spacing: 10
+            topPadding: 30
+            bottomPadding: 30
+
+            CoreText {
+                text: qsTr("Information");
+                bold: true;
+                font.pixelSize: 18;
+                horizontalAlignment: Qt.AlignLeft;
+                color: Theme.color.neutral9;
+            }
+
+            Column {
+                width: parent.width
+                bottomPadding: 5
+
+                KeyValueRow { key: KeyText { text: qsTr("Address"); } value: ValText { text: details.address; color: Theme.color.neutral9; }}
+                KeyValueRow { key: KeyText { text: qsTr("VIA"); } value: ValText { text: details.addressLocal; color: Theme.color.neutral9; }}
```

Comment:
maybe _VIA_ could be displayed only when it has a value?
for example, for an onion peer there is no VIA

Reply:
I do agree, unless there's reason for leaving it empty.

Reply:
As I saw in previous approach of the `PeerDetails` page #387, perhaps a "`N/A`" could be displayed or "`--`"? ([e.g.](https://github.com/bitcoin-core/gui-qml/pull/387#issuecomment-1943106202))